### PR TITLE
update for latest sync api changes

### PR DESCRIPTION
**Pull request by karissa**
depends upon https://github.com/digidem/mapeo-core/pull/21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,23 +160,46 @@ Fetch a single vector tile from the tileset `id` by an `x`,`y`,`z` coordinate.
...
+Returns list of available sync peers every set interval. Only lists other devices broadcasting using the 'mapeo-sync' key.
+
+Each sync target is an object with `ip`, `port`, `host`, and `name` if device
+name is set.
+
+```js
+GET /sync/peers?interval=300
+
+// then every 300ms...
+{ topic: 'peers', message: [{ip: 127.0.0.1, port: 1337, host: 'hostname', name: "Karissa's Device"}]}
```
Comment (noffle) [inline README.md]:
```suggestion
{ topic: 'peers', message: [{ip: '127.0.0.1', port: 1337, host: 'hostname', name: "Karissa's Device"}]}
```

File: README.md [hunk 2]
```diff
@@ -160,23 +160,46 @@ Fetch a single vector tile from the tileset `id` by an `x`,`y`,`z` coordinate.
...
+{ topic: 'peers', message: [{ip: 127.0.0.1, port: 1337, host: 'hostname', name: "Karissa's Device"}]}
```
Comment (noffle) [inline README.md]:
```suggestion
{ topic: 'peers', message: [{ip: '127.0.0.1', port: 1337, host: 'hostname', name: "Karissa's Device"}]}
```

**Comment by noffle** [comment]
This is a breaking change right?

**Comment by karissa** [comment]
yes it is!

**Pull request merged**